src/System.Security.Cryptography.Primitives/src/facade/System.Security.Cryptography.Primitives.csproj

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <!-- Setting default TargetGroup before importing dir.prop -->
-    <TargetGroup Condition="'$(TargetGroup)' == ''">net46</TargetGroup>
+    <Configuration Condition="'$(Configuration)'==''">net46_Debug</Configuration>
```

Comment:
This is a facade project and so the only configuration this project actually builds is net46 so this allows someone to build this project from the command line without passing any parameters. Without it if you build this project from the commandline without and parameters it will default the Configuration to Debug and AnyOS and this project will fail because it will not be able to resolve the dependencies for it.